### ISSUE TITLE
output color profile sometimes isn't honored when exporting to jpg

### ISSUE THREAD

**Issue by paolobenve** [darktable-org/darktable#6372]
Starting with 3.2.1, it happens that sometimes the exported jpg has the raw's _input color profile_ as its color profile.

All my raws (cr3's converted to dng's) have "linear REC2020 RGB" as work profile in the input color profile module (I didn't make any change), all my raws has the default, "sRGB (web-safe)" as output color profile (again, I didn't change anything).

However, some (no all; why?) of the exported jpgs have "linear REC2020 RGB" as color profile.

This produces issues when reducing, thumbnailing, ecc.

[This zip](https://benvenuto.cathopedia.org/color%20profile%20issue.zip) has two dng's with their sidecar files, one dng produces correct jpgs, the other produces incorrect ones.

EDIT: no, wait, now both produce exported jpg with incorrect color profile

**Comment by xiota** [comment]
There are two locations in which the output profile may be set.  One is set on a per image basis.  The other is in the export options.  Are both of them set correctly?

![darktable-sidebar](https://user-images.githubusercontent.com/8353098/94161320-68ead480-fe3a-11ea-9571-85fc21546672.png)
![darktable-export](https://user-images.githubusercontent.com/8353098/94161329-6ab49800-fe3a-11ea-9fc4-1391eb019a6e.png)

**Comment by paolobenve** [comment]
actually I had some Rec2020 RGB profile in the export settings.

Actually it should have happened that I changed it without realize it.

What should I put in there? image setting or sRGB?

**Comment by xiota** [comment]
TLDR: For simplicity, just set both output settings to sRGB.

In the export settings, if you select "image settings" it *should* use whatever is set for each image. If you put something else, it would over-ride the image settings. In your case, it appears the images were set for sRGB output, but the export was set for Rec2020, so Rec2020 over-rode the sRGB setting.

If you are meticulous about checking the per image settings, it would be fine to export to "image settings". If you're not sure, it's safer to set it to "sRGB". The only time I would use something else is for 16-bit TIFF output that I'd want to edit in another program. Then when done editing, I would *still* export to sRGB JPGs.

Another scenario in which it might make sense to export to a different profile is if you are absolutely certain *all* of your intended output devices (screens, printers, etc) are compatible with a different profile, and you either don't care or are certain that everything else is properly color managed. If I were using 100% Apple devices and high-gamut monitors, I would consider using an Apple Display P3 profile. but it could cause problems similar to what you've experienced for non-Apple users (eg, images shared on social media).

**Comment by paolobenve** [comment]
thank you, your explications are really good!
